examples/mssql/README.md

```diff
@@ -0,0 +1,87 @@
+### Prerequisites
+
+Docker compatible [*nix OS](https://en.wikipedia.org/wiki/Unix-like) and Docker installed.
+<br>Download this folder and give execute permission to the scripts i.e.</br>
+```bash
+wget -c https://github.com/RedisGears/rgsync/archive/master.zip && unzip master.zip "rgsync-master/examples/mssql/*" && rm master.zip && mv rgsync-master rgsync && cd rgsync/examples/mssql && chmod a+x *.sh
+```
+
+# Redis OSS Users
+Assuming you have RedisGears up and running (see [Quick Start](https://oss.redislabs.com/redisgears/quickstart.html)). Please continue from [Setup MSSQL 2017 database in docker](#setup-mssql-2017-database-in-docker).
+
+
+# Redis Enterprise Users
+
+## Setup Redis Enterprise cluster and database in docker
+<br>Execute [setupAndcreate_redb.sh](setupAndcreate_redb.sh)</br>
+```bash
+> ./setupAndcreate_redb.sh
+```
+---
+**NOTE**
+
+The above script will create a 3-node Redis Enterprise cluster in docker containers on rg-net network, [Install RedisGears dependencies](https://docs.redislabs.com/latest/modules/redisgears/installing-redisgears/#step-1-install-redisgears-dependencies), [Install RedisGears module](https://docs.redislabs.com/latest/modules/redisgears/installing-redisgears/#step-2-install-the-redisgears-module), [Install MS ODBC driver for MSSQL](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15#ubuntu17) and [Create a Redis Enterprise database with RedisGears module enabled and verify the installation](https://docs.redislabs.com/latest/modules/redisgears/installing-redisgears/#step-3-create-a-database-and-verify-the-installation).
+
+---
+
+## Setup MSSQL 2017 database in docker
```

Comment:
please move this under the Prerequisites section

Reply:
Updated. Thanks!